extension/src/paymentHandler/payment-utils.js

```diff
@@ -68,7 +68,7 @@ function createAddInterfaceInteractionAction (
     fields: {
       createdAt: new Date(),
       response: JSON.stringify(response),
-      request: JSON.stringify(request),
+      request: JSON.stringify(request.body),
```

Comment:
good question, I found a breaking change 😞I need to check the part below.

https://github.com/commercetools/commercetools-adyen-integration/blob/7cc6018016f8c15f64ec23dc04544411280c9d09/extension/src/paymentHandler/payment-utils.js#L85-L87

for your question it will have a body always, it's ensured while request creation, should I add a check for this method?

Reply:
We decided with @butenkor, it is not a problem or breaking change..